### ISSUE TITLE
Incorrect dropdown style when Bootstrap 4 version is not auto detected since 1.13.13

### ISSUE THREAD

**Issue by rjaros** [snapappointments/bootstrap-select#2448]
When Bootstrap 4 version is not auto detected and is set manually with:
```
$.fn.selectpicker.Constructor.BootstrapVersion = '4';
``` 
the dropdown style is broken, probably because of this change: https://github.com/snapappointments/bootstrap-select/commit/8698d1bff5c62be8465c895af038b520d34d8149

`version.major` is still `'3'` when the default element template is generated.

**Comment by ShirleySunCN** [comment]
@caseyjhol  I made some adjustment in my project to workaround this issue, I would like to ensure this is ok, please help to review it.
And hope for the next release, so great extension.
```
  .bootstrap-select > .btn{
    border: 1px solid #ced4da;
    background-color: white;
    width: 100%;
  }

  .bootstrap-select > .btn > div{
    float: left;
    width: 90%;
    text-align: left;
  }

  .bootstrap-select > .selectpicker {
    display: none;
  }

  .bootstrap-select > .dropdown-menu {
    padding: 0;
    max-height: 230px;
  }

  .bootstrap-select > .dropdown-menu > .inner > .dropdown-menu{
    top: 0;
    position: relative;
    width: 100%;
  }
```

Bug screen capture:
![image](https://user-images.githubusercontent.com/31336898/81066668-09823b80-8f10-11ea-9353-8355d5511c09.png)

After modified the style:
![image](https://user-images.githubusercontent.com/31336898/81066801-3e8e8e00-8f10-11ea-9a55-e1e25785f0b0.png)

**Comment by rjaros** [comment]
To workaround this issue I'm inserting this small code fragment before loading `bootstrap-select.js`:
```
if ($.fn.dropdown === undefined) { $.fn.dropdown={'Constructor' : {'VERSION' : '4.0.0'}}; };
```

**Comment by ShirleySunCN** [comment]
@raido wow, thank you so much!

**Comment by raido** [comment]
I am not sure I am tagged here correctly :).

**Comment by rjaros** [comment]
Surely not :-) You are just first in github autocomplete list when someone writes `@r` ;-)

**Comment by caseyjhol** [comment]
Released in [v1.13.17](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.17)!
